examples/anonymous-functions/anonymous_functions.description

```diff
@@ -0,0 +1,2 @@
+// Simply said, Anonymous Functions are a syntactic shortcut for defining inline functions.
+// In comparison to a normal function definition, the only missing part is the name.
```

Comment:
Shall we change as follows for clarity:
"// In Ballerina, anonymous functions are a syntactic shortcut for defining inline functions.
// When you compare an anonymous function to a normal function definition in Ballerina, the only missing part is the function name."